[pytorch] upload android build size to scuba

ljk53 opened the pull request:
Stack from [ghstack](https://github.com/ezyang/ghstack):
* **#40010 [pytorch] upload android build size to scuba**

Summary:
Use `upload_binary_size_to_scuba.py` to extract android library size and
upload to scuba.

Sample data: https://fburl.com/scuba/pytorch_binary_size/fz882auc
```
+-----------------------------------------------------------+-------------------+-----------+---------+-------------------------------------------------+------------------------------------------+----------+
|                        Build Name                         |      Branch       | Build Num |   Os    |                    Pkg Type                     |                   Sha1                   |   Size   |
+-----------------------------------------------------------+-------------------+-----------+---------+-------------------------------------------------+------------------------------------------+----------+
| linux_libtorch_3.7m_cpu                                   | gh/ljk53/149/head |   5842365 | linux   | libtorch                                        | f6821a3708bb2e88d9482f5ff60ecc6bcc60d85c | 111 MiB  |
| android_prebuild/aar/pytorch_android-release.aar__        | gh/ljk53/149/head |   5842981 | android | prebuild/aar/pytorch_android-release.aar        | f6821a3708bb2e88d9482f5ff60ecc6bcc60d85c | 29.1 MiB |
| android_prebuild-single/aar/pytorch_android-release.aar__ | gh/ljk53/149/head |   5842974 | android | prebuild-single/aar/pytorch_android-release.aar | f6821a3708bb2e88d9482f5ff60ecc6bcc60d85c | 7.96 MiB |
| android_prebuild/x86_64/libpytorch_jni.so__               | gh/ljk53/149/head |   5842981 | android | prebuild/x86_64/libpytorch_jni.so               | f6821a3708bb2e88d9482f5ff60ecc6bcc60d85c | 7.65 MiB |
| android_prebuild-single/x86/libpytorch_jni.so__           | gh/ljk53/149/head |   5842974 | android | prebuild-single/x86/libpytorch_jni.so           | f6821a3708bb2e88d9482f5ff60ecc6bcc60d85c | 7.62 MiB |
| android_prebuild/x86/libpytorch_jni.so__                  | gh/ljk53/149/head |   5842981 | android | prebuild/x86/libpytorch_jni.so                  | f6821a3708bb2e88d9482f5ff60ecc6bcc60d85c | 7.62 MiB |
| android_prebuild/arm64-v8a/libpytorch_jni.so__            | gh/ljk53/149/head |   5842981 | android | prebuild/arm64-v8a/libpytorch_jni.so            | f6821a3708bb2e88d9482f5ff60ecc6bcc60d85c | 6.44 MiB |
| android_prebuild/armeabi-v7a/libpytorch_jni.so__          | gh/ljk53/149/head |   5842981 | android | prebuild/armeabi-v7a/libpytorch_jni.so          | f6821a3708bb2e88d9482f5ff60ecc6bcc60d85c | 6.23 MiB |
| android_prebuild/x86_64/libfbjni.so__                     | gh/ljk53/149/head |   5842981 | android | prebuild/x86_64/libfbjni.so                     | f6821a3708bb2e88d9482f5ff60ecc6bcc60d85c | 319 KiB  |
| android_prebuild-single/x86/libfbjni.so__                 | gh/ljk53/149/head |   5842974 | android | prebuild-single/x86/libfbjni.so                 | f6821a3708bb2e88d9482f5ff60ecc6bcc60d85c | 319 KiB  |
| android_prebuild/x86/libfbjni.so__                        | gh/ljk53/149/head |   5842981 | android | prebuild/x86/libfbjni.so                        | f6821a3708bb2e88d9482f5ff60ecc6bcc60d85c | 319 KiB  |
| android_prebuild/arm64-v8a/libfbjni.so__                  | gh/ljk53/149/head |   5842981 | android | prebuild/arm64-v8a/libfbjni.so                  | f6821a3708bb2e88d9482f5ff60ecc6bcc60d85c | 282 KiB  |
| android_prebuild/armeabi-v7a/libfbjni.so__                | gh/ljk53/149/head |   5842981 | android | prebuild/armeabi-v7a/libfbjni.so                | f6821a3708bb2e88d9482f5ff60ecc6bcc60d85c | 214 KiB  |
+-----------------------------------------------------------+-------------------+-----------+---------+-------------------------------------------------+------------------------------------------+----------+
```

Differential Revision: [D22040439](https://our.internmc.facebook.com/intern/diff/D22040439)

ljk53 committed:
[pytorch] upload android build size to scuba

Summary:
Use `upload_binary_size_to_scuba.py` to extract android library size and
upload to scuba.

[ghstack-poisoned]

ljk53 committed:
Update on "[pytorch] upload android build size to scuba"


Summary:
Use `upload_binary_size_to_scuba.py` to extract android library size and
upload to scuba.

[ghstack-poisoned]

ljk53 committed:
Update on "[pytorch] upload android build size to scuba"


Summary:
Use `upload_binary_size_to_scuba.py` to extract android library size and
upload to scuba.

[ghstack-poisoned]

ljk53 committed:
Update on "[pytorch] upload android build size to scuba"


Summary:
Use `upload_binary_size_to_scuba.py` to extract android library size and
upload to scuba.

[ghstack-poisoned]

ljk53 committed:
Update on "[pytorch] upload android build size to scuba"


Summary:
Use `upload_binary_size_to_scuba.py` to extract android library size and
upload to scuba.

[ghstack-poisoned]

ljk53 committed:
Update on "[pytorch] upload android build size to scuba"


Summary:
Use `upload_binary_size_to_scuba.py` to extract android library size and
upload to scuba.

[ghstack-poisoned]

ljk53 committed:
Update on "[pytorch] upload android build size to scuba"


Summary:
Use `upload_binary_size_to_scuba.py` to extract android library size and
upload to scuba.

[ghstack-poisoned]

ljk53 committed:
Update on "[pytorch] upload android build size to scuba"


Summary:
Use `upload_binary_size_to_scuba.py` to extract android library size and
upload to scuba.

Sample data: https://fburl.com/scuba/pytorch_binary_size/fz882auc
```
+-----------------------------------------------------------+-------------------+-----------+---------+-------------------------------------------------+------------------------------------------+----------+
|                        Build Name                         |      Branch       | Build Num |   Os    |                    Pkg Type                     |                   Sha1                   |   Size   |
+-----------------------------------------------------------+-------------------+-----------+---------+-------------------------------------------------+------------------------------------------+----------+
| linux_libtorch_3.7m_cpu                                   | gh/ljk53/149/head |   5842365 | linux   | libtorch                                        | f6821a3708bb2e88d9482f5ff60ecc6bcc60d85c | 111 MiB  |
| android_prebuild/aar/pytorch_android-release.aar__        | gh/ljk53/149/head |   5842981 | android | prebuild/aar/pytorch_android-release.aar        | f6821a3708bb2e88d9482f5ff60ecc6bcc60d85c | 29.1 MiB |
| android_prebuild-single/aar/pytorch_android-release.aar__ | gh/ljk53/149/head |   5842974 | android | prebuild-single/aar/pytorch_android-release.aar | f6821a3708bb2e88d9482f5ff60ecc6bcc60d85c | 7.96 MiB |
| android_prebuild/x86_64/libpytorch_jni.so__               | gh/ljk53/149/head |   5842981 | android | prebuild/x86_64/libpytorch_jni.so               | f6821a3708bb2e88d9482f5ff60ecc6bcc60d85c | 7.65 MiB |
| android_prebuild-single/x86/libpytorch_jni.so__           | gh/ljk53/149/head |   5842974 | android | prebuild-single/x86/libpytorch_jni.so           | f6821a3708bb2e88d9482f5ff60ecc6bcc60d85c | 7.62 MiB |
| android_prebuild/x86/libpytorch_jni.so__                  | gh/ljk53/149/head |   5842981 | android | prebuild/x86/libpytorch_jni.so                  | f6821a3708bb2e88d9482f5ff60ecc6bcc60d85c | 7.62 MiB |
| android_prebuild/arm64-v8a/libpytorch_jni.so__            | gh/ljk53/149/head |   5842981 | android | prebuild/arm64-v8a/libpytorch_jni.so            | f6821a3708bb2e88d9482f5ff60ecc6bcc60d85c | 6.44 MiB |
| android_prebuild/armeabi-v7a/libpytorch_jni.so__          | gh/ljk53/149/head |   5842981 | android | prebuild/armeabi-v7a/libpytorch_jni.so          | f6821a3708bb2e88d9482f5ff60ecc6bcc60d85c | 6.23 MiB |
| android_prebuild/x86_64/libfbjni.so__                     | gh/ljk53/149/head |   5842981 | android | prebuild/x86_64/libfbjni.so                     | f6821a3708bb2e88d9482f5ff60ecc6bcc60d85c | 319 KiB  |
| android_prebuild-single/x86/libfbjni.so__                 | gh/ljk53/149/head |   5842974 | android | prebuild-single/x86/libfbjni.so                 | f6821a3708bb2e88d9482f5ff60ecc6bcc60d85c | 319 KiB  |
| android_prebuild/x86/libfbjni.so__                        | gh/ljk53/149/head |   5842981 | android | prebuild/x86/libfbjni.so                        | f6821a3708bb2e88d9482f5ff60ecc6bcc60d85c | 319 KiB  |
| android_prebuild/arm64-v8a/libfbjni.so__                  | gh/ljk53/149/head |   5842981 | android | prebuild/arm64-v8a/libfbjni.so                  | f6821a3708bb2e88d9482f5ff60ecc6bcc60d85c | 282 KiB  |
| android_prebuild/armeabi-v7a/libfbjni.so__                | gh/ljk53/149/head |   5842981 | android | prebuild/armeabi-v7a/libfbjni.so                | f6821a3708bb2e88d9482f5ff60ecc6bcc60d85c | 214 KiB  |
+-----------------------------------------------------------+-------------------+-----------+---------+-------------------------------------------------+------------------------------------------+----------+
```

[ghstack-poisoned]

ezyang commented:
This looks vaguely reasonable but I'm going to let some other folks on the reviewer list take a crack at it.

ljk53 committed:
Update on "[pytorch] upload android build size to scuba"


Summary:
Use `upload_binary_size_to_scuba.py` to extract android library size and
upload to scuba.

Sample data: https://fburl.com/scuba/pytorch_binary_size/fz882auc
```
+-----------------------------------------------------------+-------------------+-----------+---------+-------------------------------------------------+------------------------------------------+----------+
|                        Build Name                         |      Branch       | Build Num |   Os    |                    Pkg Type                     |                   Sha1                   |   Size   |
+-----------------------------------------------------------+-------------------+-----------+---------+-------------------------------------------------+------------------------------------------+----------+
| linux_libtorch_3.7m_cpu                                   | gh/ljk53/149/head |   5842365 | linux   | libtorch                                        | f6821a3708bb2e88d9482f5ff60ecc6bcc60d85c | 111 MiB  |
| android_prebuild/aar/pytorch_android-release.aar__        | gh/ljk53/149/head |   5842981 | android | prebuild/aar/pytorch_android-release.aar        | f6821a3708bb2e88d9482f5ff60ecc6bcc60d85c | 29.1 MiB |
| android_prebuild-single/aar/pytorch_android-release.aar__ | gh/ljk53/149/head |   5842974 | android | prebuild-single/aar/pytorch_android-release.aar | f6821a3708bb2e88d9482f5ff60ecc6bcc60d85c | 7.96 MiB |
| android_prebuild/x86_64/libpytorch_jni.so__               | gh/ljk53/149/head |   5842981 | android | prebuild/x86_64/libpytorch_jni.so               | f6821a3708bb2e88d9482f5ff60ecc6bcc60d85c | 7.65 MiB |
| android_prebuild-single/x86/libpytorch_jni.so__           | gh/ljk53/149/head |   5842974 | android | prebuild-single/x86/libpytorch_jni.so           | f6821a3708bb2e88d9482f5ff60ecc6bcc60d85c | 7.62 MiB |
| android_prebuild/x86/libpytorch_jni.so__                  | gh/ljk53/149/head |   5842981 | android | prebuild/x86/libpytorch_jni.so                  | f6821a3708bb2e88d9482f5ff60ecc6bcc60d85c | 7.62 MiB |
| android_prebuild/arm64-v8a/libpytorch_jni.so__            | gh/ljk53/149/head |   5842981 | android | prebuild/arm64-v8a/libpytorch_jni.so            | f6821a3708bb2e88d9482f5ff60ecc6bcc60d85c | 6.44 MiB |
| android_prebuild/armeabi-v7a/libpytorch_jni.so__          | gh/ljk53/149/head |   5842981 | android | prebuild/armeabi-v7a/libpytorch_jni.so          | f6821a3708bb2e88d9482f5ff60ecc6bcc60d85c | 6.23 MiB |
| android_prebuild/x86_64/libfbjni.so__                     | gh/ljk53/149/head |   5842981 | android | prebuild/x86_64/libfbjni.so                     | f6821a3708bb2e88d9482f5ff60ecc6bcc60d85c | 319 KiB  |
| android_prebuild-single/x86/libfbjni.so__                 | gh/ljk53/149/head |   5842974 | android | prebuild-single/x86/libfbjni.so                 | f6821a3708bb2e88d9482f5ff60ecc6bcc60d85c | 319 KiB  |
| android_prebuild/x86/libfbjni.so__                        | gh/ljk53/149/head |   5842981 | android | prebuild/x86/libfbjni.so                        | f6821a3708bb2e88d9482f5ff60ecc6bcc60d85c | 319 KiB  |
| android_prebuild/arm64-v8a/libfbjni.so__                  | gh/ljk53/149/head |   5842981 | android | prebuild/arm64-v8a/libfbjni.so                  | f6821a3708bb2e88d9482f5ff60ecc6bcc60d85c | 282 KiB  |
| android_prebuild/armeabi-v7a/libfbjni.so__                | gh/ljk53/149/head |   5842981 | android | prebuild/armeabi-v7a/libfbjni.so                | f6821a3708bb2e88d9482f5ff60ecc6bcc60d85c | 214 KiB  |
+-----------------------------------------------------------+-------------------+-----------+---------+-------------------------------------------------+------------------------------------------+----------+
```

Differential Revision: [D22040439](https://our.internmc.facebook.com/intern/diff/D22040439)

[ghstack-poisoned]
